inputters/base.lua

```diff
@@ -33,22 +33,11 @@ end
 
 function inputter:requireClass (tree)
   local root = SILE.documentState.documentClass == nil
-  local document
   if root then
-    for _, leaf in ipairs(tree) do
-      if leaf.lno then
-        if document then
-          SU.error("Document has more than one parent node!")
-        end
-        if leaf.command == "sile" or leaf.command == "document" then
-          document = leaf
-        end
-      end
-    end
-    if not document then
+    if tree.command ~= "sile" and tree.command ~= "document" then
       SU.error("This isn't a SILE document!")
     end
-    self:classInit(document.options or {})
+    self:classInit(tree.options or {})
```

Comment:
So compared to 0.14.4, this is acting on `tree` now (vs. `tree[1]` before).

Does this mean I need to change:
https://github.com/Omikhleia/markdown.sile/blob/53851473b631af24d9f1a008ce524c08a13015f4/inputters/markdown.lua#L354

From (working in 0.14.4)
```
tree = { { tree, command = "document", options = { class = "markdown" } } }
```
To (maybe working on 0.14.x, x > 5)
```
tree = { tree, command = "document", options = { class = "markdown" } }
```
?

Reply:
No, I'm reverting this. It was/is a mechanism for passing multiple documents in the future.